store/driver/tikv_driver.go

```diff
@@ -235,7 +235,7 @@ func (d TiKVDriver) OpenWithOptions(path string, options ...Option) (resStore kv
 		tikv.WithCodec(codec),
 	)
 
-	s, err = tikv.NewKVStore(uuid, pdClient, spkv, rpcClient, tikv.WithPDHTTPClient(tlsConfig, etcdAddrs))
+	s, err = tikv.NewKVStore(uuid, pdClient, spkv, rpcClient)
```

Comment:
Can you explain why `tikv.WithPDHTTPClient(tlsConfig, etcdAddrs)` is the trouble maker?

Reply:
It enables the PD http client to get store resolved ts